Sources/Paywalls/Components/Transitions/PaywallAnimation.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PaywallAnimation.swift
+//
+//  Created by Jacob Zivan Rakidzich on 8/20/25.
+// swiftlint:disable missing_docs
+
+import Foundation
+
+public extension PaywallComponent {
+    struct Animation: PaywallComponentBase {
```

Comment:
[nitpick] We normally have one empty line after type declarations, and one before the end of it https://github.com/RevenueCat/purchases-ios/blob/d98a15aa82cf6556eec4a1f0e45934edc868ea8c/Contributing/SwiftStyleGuide.swift#L18